c/example-4-piggybank/README.md

```diff
@@ -0,0 +1,49 @@
+# Piggybank
+
+This contract has two methods. The first method (`initialize_target`)
```

Comment:
This contract has two methods. The first method (`initialize_target`)
specifies the target amount of money that should be collected by
the contract.

The second method (`transfer`) transfers almost all the money from 
the account (reduced by fixed `MESSAGE_COST` amount, a fixed constant in
the contract C file) when the contract accumulates the target amount of money or more.